app/services/manifest_validator.rb

```diff
@@ -0,0 +1,28 @@
+class ManifestValidator
+  include JsonSchemaHelper
+
+  attr_reader :content, :errors
```

Comment:
they are exercised in the unit tests, which validate that the internals work as expected. I.e. that you can pass a String or Hash into the constructor, and get the full error stack back out after `run`.